src/jsp/com/cubrid/jsp/data/DBType.java

```diff
@@ -0,0 +1,54 @@
+/*
+ *
+ * Copyright (c) 2016 CUBRID Corporation.
+ *
+ * Redistribution and use in source and binary forms, with or without modification,
+ * are permitted provided that the following conditions are met:
+ *
+ * - Redistributions of source code must retain the above copyright notice,
+ *   this list of conditions and the following disclaimer.
+ *
+ * - Redistributions in binary form must reproduce the above copyright notice,
+ *   this list of conditions and the following disclaimer in the documentation
+ *   and/or other materials provided with the distribution.
+ *
+ * - Neither the name of the <ORGANIZATION> nor the names of its contributors
+ *   may be used to endorse or promote products derived from this software without
+ *   specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+ * IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA,
+ * OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+ * WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY
+ * OF SUCH DAMAGE.
+ *
+ */
+
+package com.cubrid.jsp.data;
+
+public class DBType {
+    public static final int DB_NULL = 0;
+    public static final int DB_INT = 1;
+    public static final int DB_FLOAT = 2;
+    public static final int DB_DOUBLE = 3;
+    public static final int DB_STRING = 4;
+    public static final int DB_OBJECT = 5;
+    public static final int DB_SET = 6;
+    public static final int DB_MULTISET = 7;
+    public static final int DB_SEQUENCE = 8;
+    public static final int DB_TIME = 10;
+    public static final int DB_TIMESTAMP = 11;
+    public static final int DB_DATE = 12;
+    public static final int DB_MONETARY = 13;
+    public static final int DB_SHORT = 18;
+    public static final int DB_NUMERIC = 22;
+    public static final int DB_CHAR = 25;
+    public static final int DB_RESULTSET = 28;
+    public static final int DB_BIGINT = 31;
+    public static final int DB_DATETIME = 32;
+}
```

Comment:
Is  JSON type not supported?

Reply:
As far as I know, JSON is considered as String type.